plugins/performance-lab/includes/site-health/effective-asset-cache-headers/hooks.php

```diff
@@ -22,10 +22,21 @@
  * @return array{direct: array<string, array{label: string, test: string}>} Amended tests.
  */
 function perflab_effective_asset_cache_headers_add_test( array $tests ): array {
+	/*
+	 * Static assets are expected to not have effective cache headers in non-production environments.
+	 * Hence bail early.
+	 *
+	 * GH Issue: https://github.com/WordPress/performance/issues/2031
+	 */
+	if ( in_array( wp_get_environment_type(), array( 'local', 'development' ), true ) ) {
+		return $tests;
+	}
+
 	$tests['direct']['effective_asset_cache_headers'] = array(
 		'label' => __( 'Effective Caching Headers', 'performance-lab' ),
 		'test'  => 'perflab_effective_asset_cache_headers_assets_test',
 	);
+
```

Comment:
I think this is _slightly_ better as it means there is only one `return $tests`:

```suggestion
	if ( ! in_array( wp_get_environment_type(), array( 'local', 'development' ), true ) ) {
		$tests['direct']['effective_asset_cache_headers'] = array(
			'label' => __( 'Effective Caching Headers', 'performance-lab' ),
			'test'  => 'perflab_effective_asset_cache_headers_assets_test',
		);
	}
```

Reply:
Okay, would update this to use only 1 return statement.

Reply:
This is now fixed and function is updated.

Thank You,